README.md

```diff
@@ -19,10 +19,11 @@ The goal of this specification is to enable the creation of interoperable tools
 
 ## Overview
 
-This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md) and may with a [manifest list](manifest-list.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
```

Comment:
“and may with” reads awkwardly to me.  I'd rather leave this line alone, land your new “The manifest list” line, and then land a rerolled #456 mentioning manifest lists.